website/server/middlewares/cron.js

```diff
@@ -6,9 +6,10 @@ import Bluebird from 'bluebird';
 import { model as Group } from '../models/group';
 import { model as User } from '../models/user';
 import { recoverCron, cron } from '../libs/cron';
-import { v4 as uuid } from 'uuid';
 
 const daysSince = common.daysSince;
+// Wait this length of time in ms before attempting another cron
+const cronTimeout = new Date(3600000).getTime();
```

Comment:
Name changed. I kept the comment the same so that it will be accurate if a different time is selected in the future